mythical-creatures/exercises/dragon.js

```diff
@@ -0,0 +1,21 @@
+class Dragon {
+  constructor(name, rider, color) {
+    this.name = name;
+    this.rider = rider;
+    this.color = color;
+    this.hungry = true;
+  }
+
+  eat() {
```

Comment:
Yes - just like you said in the PR comment - here you would have to use some kind of `count` variable so you know how many times you've been in this function, aka how many times the dragon has eaten.